modules/dists/ReplicatedDist.chpl

```diff
@@ -62,79 +59,29 @@ That is, changes to one replicand of an array are never propagated to
 the other replicands by the distribution implementation.
 If desired, consistency needs to be maintained by the user.
 
-Replication over locales is observable when:
-
-* iterating over a domain or array
-
-* printing with ``writeln()`` et al.
-
-* zippering, when the replicated domain/array is
-  the first among the zippered items
-
-* assigning into the replicated array
-  (each replicand gets a copy)
-
-* inquiring about the domain's ``numIndices``
-  or the array's ``numElements``
-
-* accessing array element(s) from a locale *not* in the set of desired locales,
-  i.e. from a locale which the array is not replicated onto.
-  Upon such an access, an out-of-bounds error is reported.
-
-Only the replicand *on the current locale* is accessed
-(i.e. existence of multiple replicands is not observable) when:
-
-* examining certain domain properties:
-  ``dim(d)``, ``dims()``, ``low``, ``high``, ``stride``
-  -- not ``numIndices``
-
-* indexing into an array
-
-* zippering, when the first zippered item is not replicated
-
-* assigning to a non-replicated array,
-  i.e. the replicated array is on the right-hand side of the assignment
-
-* there is only a single locale
-  (trivially: there is only one replicand in this case)
-
-.. when slicing an array?
-
-E.g. when iterating, the number of iterations will be (the number of
-locales involved) times (the number of iterations over this domain if
-it were distributed with the default distribution).
-
-Note that the above behavior may change in the future. In particular,
-we are considering changing it so that replication is never observable.
-For example, only the local replicand would be accessed in all cases.
+When accessing a replicated domain or array from a locale *not* in the
+set of target locales, an error is reported if bounds-checking is on;
+undefined behavior occurs if it is off.
 
+Otherwise, only the replicand on the current locale is accessed by
+default when performing operations on the domain or array.
```

Comment:
Is "by default" meaningful here? If yes, as a user I'd like to know how to change from the default. If no, let's take that out.